cmake/thirdparty/get_tblis.cmake

```diff
@@ -136,20 +143,21 @@ function(find_or_configure_tblis)
 
     # Makes `target_include_directories()` below work
     file(MAKE_DIRECTORY "${tblis_BINARY_DIR}/include")
+
+    add_library(tblis::tblis ALIAS tblis)
```

Comment:
This should be inside the IF statement since the `rapids_find_generate_module` will generate the target if found.